src/js/index.js

```diff
@@ -225,11 +228,47 @@ export const hpe = deepFreeze({
   checkBox: {
     gap: 'medium',
     color: 'text-strong',
+    toggle: {
+      color: {
+        dark: global.colors['text-strong'].dark,
+        light: global.colors['text-strong'].dark,
```

Comment:
should both dark+light use the dark color? it doesn't feel right.

Reply:
This matches what is in the sticker sheet -- it means that the toggle knob (when not checked) is always #FFFFFF.

Reply:
Resolved via slack -- knob should use background/background front color